.github/workflows/update-viablestrict.yml

```diff
@@ -0,0 +1,88 @@
+name: Update viable/strict
+
+on:
+  workflow_call:
+    inputs:
+      repository:
+        description: 'Repository to checkout, defaults to "".'
+        default: ""
+        type: string
+      test-infra-repository:
+        description: "Test infra repository to use/"
+        default: "pytorch/test-infra"
+        type: string
+      test-infra-ref:
+        description: "Test infra reference to use."
+        default: ""
+        type: string
+      required_checks:
+        description: "A comma-separated string of workflows to be completed successfully"
+        required: true
+        type: string
+      viable_strict_branch:
+        description: "The name of the viable/strict branch."
+        default: "viable/strict"
+        type: string
+    secrets:
+      ROCKSET_API_KEY:
+        required: true
+        description: 'API key for ROCKSET'
+      GITHUB_DEPLOY_KEY:
+        required: true
+        description: 'GitHub deploy key'
+
+
+jobs:
+  do_update_viablestrict:
+    runs-on: ubuntu-20.04
```

Comment:
Looks like this is a GH-hosted runner?

Reply:
I think that should be fine in this context